clients/src/main/java/org/apache/kafka/clients/consumer/internals/ClassicKafkaConsumer.java

```diff
@@ -265,7 +266,7 @@ public class ClassicKafkaConsumer<K, V> implements ConsumerDelegate<K, V> {
             // call close methods if internal objects are already constructed; this is to prevent resource leak. see KAFKA-2121
             // we do not need to call `close` at all when `log` is null, which means no internal objects were initialized.
             if (this.log != null) {
-                close(Duration.ZERO, true);
+                close(Duration.ZERO, CloseOptions.GroupMembershipOperation.DEFAULT, true);
```

Comment:
ditto here: hardcode to `REMAIN_IN_GROUP`?

Reply:
Make sense. We could prevent from sending a leave request here. 
I will also add a comment to avoid confusion.